packages/core/src/definitions/styles.ts

```diff
@@ -193,7 +193,7 @@ export const optionalBuiltInStyles: VariableDefinitions = {
       width: DEFAULT_IMAGE_WIDTH,
       height: '100%',
       objectPosition: 'center center',
-      quality: '100',
+      quality: '80%',
```

Comment:
Does this mean that images will default to 80% quality when the image optimization section in the UI is "collapsed"?  

We might need a UI change to incorporate the 80% so that we still default images to 100% quality, and only set the "initial value" to 80% when the image optimization section is expanded/enabled in the UI.

Reply:
The desired default quality is 80%, is that still an issue in the UI?

Reply:
I thought we wanted the quality to be 100% by default, and then set quality to 80% as the "initial" value only when image optimization is enabled.

If the default quality should be 80% even when image optimization is disabled this should accomplish that without any change needed to the UI

Reply:
Quality should be not set by default (which having it not set is the same as 100% practically), and 80% when image optimization is enabled.

I do have a PR incoming for the UI as I ran into some other issues, I'll send that over soonish.

Reply:
Awesome work on these updates! 🎉 

This makes sense to start some options as not set by default.  Your UI changes are looking great, really cleans up the overall implementation quite a bit! 👏